<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela</class>
 <widget class="QMainWindow" name="janela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Arabic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Programa Brinquedoteca</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>341</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>foto 2.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nome da Crian√ßa</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Idade</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="idade">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_adicionar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background: green;
color: white</string>
    </property>
    <property name="text">
     <string>Adicionar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_listar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background: blue;
color: white</string>
    </property>
    <property name="text">
     <string>Listar Todos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_limpar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>560</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background: red;
color: white</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fechar">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>560</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background: rgb(74, 74, 74);
color: white</string>
    </property>
    <property name="text">
     <string>Fechar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="nome">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="filtro">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Filtro</string>
    </property>
    <item>
     <property name="text">
      <string>Filtro</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
